.gitignore

```diff
@@ -6,3 +6,7 @@
 /sv2.h
 /test/bitcoin_data/regtest
 lcov.info
+
+
+/roles/v2/pool/pool-config.toml
```

Comment:
Why do we want to remove these example configs from the repo?

Reply:
left them for test purpose. I'll delete these lines